.platform.app.yaml

```diff
@@ -18,14 +22,16 @@ hooks:
   build: |
     set -e
     echo "Installing Swoole extension"
-    bash install-swoole.sh v4.6.2
+    bash install-swoole.sh openswoole 4.11.1
+    echo "Current PHP modules:"
+    php -m
     echo "Installing platform.sh CLI"
     curl -sS https://platform.sh/cli/installer | php
     echo "- Setting up configuration"
     rm -f config/development.config.php config/autoload/*.local.php
     mv config/autoload/local.php.dist config/autoload/local.php
     echo "- Installing application dependencies"
-    composer install --ignore-platform-reqs --no-ansi --no-dev --no-interaction --no-scripts --no-plugins --optimize-autoloader
+    composer install --no-ansi --no-dev --no-interaction --no-scripts --no-plugins --optimize-autoloader --ignore-platform-req=ext-openswoole
```

Comment:
This was necessary as the build env is not the same as the deployment env, and it does not honor the extension enablement done within the install-swoole.sh script.